app/helpers/tutoring_session_export_helper.rb

```diff
@@ -16,17 +16,26 @@ def create_csv(start_date, end_date, filepath = 'public/tutoring_hours.csv')
   end
 
   def mail_csv(start_date, end_date, filepath)
+    code = code_check(filepath)
     if Rails.env.test?
-      HourCheckMailer.with(begin: start_date, end: end_date, filepath: filepath,
+      HourCheckMailer.with(begin: start_date, end: end_date, code: code,
                            email: 'admin@tamu.edu').hours_email.deliver_now
     else
       HourCheckMailer.with(begin: start_date, end: end_date,
-                           filepath: filepath).hours_email.deliver_later
+                           code: code).hours_email.deliver_later
     end
   end
 
   private
 
+  def code_check(filepath)
+    code = 0
+    code = 3 unless filepath.at(/.*_3.csv/).nil?
+    code = 2 unless filepath.at(/.*_2.csv/).nil?
+    code = 1 unless filepath.at(/.*spec.csv/).nil?
+    code
+  end
```

Comment:
perhaps a comment explaining what this is would be nice

Reply:
I added a comment in the latest push, does that suffice?

Reply:
Hold on, I realized I added it to the wrong file.

Reply:
Now it's been added properly

Reply:
thanks!